### PR TITLE
Add header content area in home template.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@jgmize r?

File: masterfirefoxos/base/templates/home.html [hunk 1]
```diff
@@ -28,6 +28,10 @@ <h2 class="site-logo">
     {% endblock %}
 
       <div class="page-intro">
+        {{ render_region(feincms_page, "header", request) }}
+      </div>
+
+      <div class="page-intro">
         {{ render_region(feincms_page, "main", request) }}
```
Comment (jgmize) [inline masterfirefoxos/base/templates/home.html]: Shouldn't the main and header divs have different classes?

**Comment by craigcook** [comment]
So late yesterday we figured out we probably don't need a separate content region for the home page intro, we can just use the main content region as the intro and the rest of the page is rendered dynamically from the page's children. This assumes that the first richtext field on each child page is the description that should also appear on the home page. 

If that approach won't work for some reason then we might still need separate regions and a special content type for those home page entries, but for now I think this PR isn't needed. Sorry!

File: masterfirefoxos/base/templates/home.html (hunk 1 above)
Comment (glogiotatidis) [inline masterfirefoxos/base/templates/home.html]: Yes, maybe, i don't know. I just added the region there then Craig can do his magic.

**Comment by glogiotatidis** [comment]
Cool that can work too. I'm closing this one and we can re-open if needed.

**Pull request closed without merging**